### PR TITLE
Bugfix/composite indexes

### LINKED ISSUE #12060: [Bug]: Composite indices failing to save for relation data type
### Expected behavior

It should be possible to also create composite indices on relation data types like Many-To-One.

### Actual behavior

Currently it is not possible to create such composite keys. An error message like ```An exception occurred while executing 'ALTER TABLE `object_query_test` ADD INDEX `c_test` (test1_id,test2_id);': SQLSTATE[42000]: Syntax error or access violation: 1072 Key column 'test1_id' doesn't exist in table``` appears.

The actual fields in the database are `test1_id__id` and `test2_id__id`. 

### Steps to reproduce

1. Create a new data object class.
2. Add two Many-To-One relations to the class.
3. Try to set a composite index on these fields.
4. Error message appears on saving.

Observed in 10.4.0 with MariaDB 10.7.3

### PR DESCRIPTION
**Pull request by torqdev**
## Changes in this pull request  
Resolves #12060

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js [hunk 1]
```diff
@@ -1083,22 +1083,30 @@ pimcore.object.classes.klass = Class.create({
             value: data.index_key
         };
 
+        //fixes data to match store model
+        var indexesArray = [];
```
Comment (robertSt7) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js]:
```suggestion
        let indexesArray = [];
```

Comment (jdreesen) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js]: Why not `const`?
```suggestion
        const indexesArray = [];
```

Comment (robertSt7) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js]: @jdreesen We can also use `const`

Comment (robertSt7) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js]: @torqdev Could you please take a look at this? Thanks

Comment (jdreesen) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js]: @torqdev you know that you can just accept the suggestion, right? There's no need to manually create a commit ;)

**Comment by robertSt7** [comment]
@torqdev Thanks for the PR

**Pull request merged**